docs/setup.md

```diff
@@ -30,7 +30,7 @@ steps described in [Docker setup](#docker_hub) automatically.
 2.  Download and run the installation script:
 
     ```shell-session
-    $ bash -c "$(curl -L https://raw.githubusercontent.com/paperless-ngx/paperless-ngx/main/install-paperless-ngx.sh)"
+    $ bash -c "$(curl -L -s -S https://raw.githubusercontent.com/paperless-ngx/paperless-ngx/main/install-paperless-ngx.sh)"
```

Comment:
Want to make these curl options the long ones?  I assume a user is just going to copy them, and it might make them feel better reading the names, rather than shorthand.
 - --show-error
 - --silent
 - --location

Reply:
Sure, that makes sense. I've made the change.